### DISCUSSION TITLE
Kafka Not Restarted after kubectl annotate

### DISCUSSION THREAD

**Discussion by bj9306** [strimzi/strimzi-kafka-operator#5233] [General]
I tried to restart kafka using below command and i dont see kafka pods is restarted.

kubectl annotate statefulset watchtower-kafka strimzi.io/manual-rolling-update=true -n kafka

i see below from sts describe output

 Annotations:      kubectl.kubernetes.io/restartedAt: 2021-07-01T00:36:56-05:00

but pods are not restarted.

bj9306@TXCDTL59BT9306:~$ gp kafka
NAME                                          READY   STATUS    RESTARTS   AGE   IP           NODE                                NOMINATED NODE   READINESS GATES
watchtower-entity-operator-6dc9cc58fc-bsq4w   3/3     Running   0          16d   10.0.8.105   aks-userpool1-22444732-vmss000001   <none>           <none>
watchtower-kafka-0                            1/1     Running   0          16d   10.0.8.115   aks-userpool1-22444732-vmss000003   <none>           <none>
watchtower-kafka-1                            1/1     Running   0          16d   10.0.8.177   aks-userpool1-22444732-vmss000008   <none>           <none>
watchtower-kafka-2                            1/1     Running   0          16d   10.0.9.63    aks-userpool1-22444732-vmss00000f   <none>           <none>
watchtower-kafka-3                            1/1     Running   0          16d   10.0.9.62    aks-userpool1-22444732-vmss00000f   <none>           <none>
watchtower-kafka-4                            1/1     Running   0          89m   10.0.8.174   aks-userpool1-22444732-vmss000008   <none>           <none>
watchtower-kafka-exporter-866b4cf9c9-lklv5    1/1     Running   1          15d   10.0.8.84    aks-userpool1-22444732-vmss000001   <none>           <none>
watchtower-zookeeper-0                        1/1     Running   1          16d   10.0.8.182   aks-userpool1-22444732-vmss000008   <none>           <none>
watchtower-zookeeper-1                        1/1     Running   0          67m   10.0.8.101   aks-userpool1-22444732-vmss000001   <none>           <none>
watchtower-zookeeper-2                        1/1     Running   0          16d   10.0.8.54    aks-userpool1-22444732-vmss000000   <none>           <none>

**Comment by scholzj** [comment]
The annotation is not seen immediately, but only during the next reconciliation. If it wasn't rolled even after some time, you should probably check the operator log to see what it is doing and why is it not rolling it.
